#!/usr/bin/env python
PACKAGE = "object_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lower_bound_hue",    int_t,    0, "Lower bound hue", 0, 0, 255)
gen.add("upper_bound_hue",    int_t,    0, "Upper bound hue", 255, 0, 255)

gen.add("lower_bound_sat",    int_t,    0, "Lower bound saturation", 0, 0, 255)
gen.add("upper_bound_sat",    int_t,    0, "Upper bound saturation", 255, 0, 255)

gen.add("lower_bound_val",    int_t,    0, "Lower bound value", 0, 0, 255)
gen.add("upper_bound_val",    int_t,    0, "Upper bound value", 255, 0, 255)

exit(gen.generate(PACKAGE, "color_calibrator_node", "ColorBounds"))
